### PR TITLE
Docker Desktop Enterprise 2.3.0.0 changelogs

### PR DESCRIPTION
**Pull request by mat007**
### Proposed changes

Added Docker desktop enterprise 2.3.0.0 changelogs.

### CODE REVIEW

File: desktop/enterprise/release-notes.md [hunk 1]
```diff
@@ -17,12 +17,26 @@ For information on system requirements, installation, and download, see:
 
 For Docker Enterprise Engine release notes, see [Docker Engine release notes](/engine/release-notes).
 
+## Version 2.3.0.0
+2020-04-22
+
+Docker Desktop Enterprise 2.3.0.0 contains a Kubernetes upgrade. Note that your local Kubernetes cluster in Version Pack 3.0 will be reset after installing this version.
+
+This version contains all the bug fixes and minor changes between Docker Desktop Community versions 2.1.0.1 and 2.3.0.0 included, see:
```
Comment (stephen-turner) [inline desktop/enterprise/release-notes.md:25]: Minor rephrase: "This version includes all the bug fixes and minor changes between Docker Desktop Community versions 2.1.0.1 and 2.3.0.0. For details see:"

**Comment by usha-mandya** [comment]
LGTM with the minor suggestion noted above by Stephen.

File: desktop/enterprise/release-notes.md (hunk 1 above)
Comment (mat007) [reply desktop/enterprise/release-notes.md]: Done, thanks!

**Comment by usha-mandya** [comment]
Marked the PR as draft to avoid any accidental merge to master.

**Comment by mat007** [comment]
@usha-mandya @stephen-turner can you please take another look as I have added a commit to emphasize the manual steps required when update Docker Desktop Enterprise.
Thanks!

File: desktop/enterprise/release-notes.md [hunk 2]
```diff
@@ -17,12 +17,35 @@ For information on system requirements, installation, and download, see:
...
+
+### Bug fixes and minor changes
+
+- Windows: in the `admin-settings.json` configuration file, the `sharedDrives` and `sharedFolders` fields have been replaced by a unique `filesharingDirectories` field.
+- Mac: in the `admin-settings.json` configuration file, the `filesharingDirectories` has been moved out of the `linuxVM` section.
+- The `admin-settings.json` configuration file `configurationFileVersion` field value has been increased from 1 to 2.
+
+### Known issue
+
+Docker Desktop Enterprise 2.3.0.0 contains a configuration breaking change. Manual steps are required to update the `admin-settings.json` file, see the previous section for the list of changes to perform.
```
Comment (usha-mandya) [inline desktop/enterprise/release-notes.md:37]: I'd replace `see the previous section..` with 'see the **Bug fixes and minor changes** section..'

Comment (mat007) [reply desktop/enterprise/release-notes.md]: Done!
I assumed the bold text was on purpose and did the same.

File: desktop/enterprise/release-notes.md [hunk 3]
```diff
@@ -27,9 +27,18 @@ This version includes all the bug fixes and minor changes between Docker Desktop
...
 ### Bug fixes and minor changes
+
 - Windows: in the `admin-settings.json` configuration file, the `sharedDrives` and `sharedFolders` fields have been replaced by a unique `filesharingDirectories` field.
 - Mac: in the `admin-settings.json` configuration file, the `filesharingDirectories` has been moved out of the `linuxVM` section.
-- The `admin-settings.json` configuration file version number has been increased from 1 to 2. 
+- The `admin-settings.json` configuration file `configurationFileVersion` field value has been increased from 1 to 2.
+
+### Known issue
+
+Docker Desktop Enterprise 2.3.0.0 contains a configuration breaking change. Manual steps are required to update the `admin-settings.json` file, see the **Bug fixes and minor changes** section for the list of changes to perform.
```
Comment (stephen-turner) [inline desktop/enterprise/release-notes.md:37]: I don't like the word "breaking", it sounds too scary. I would say "contains a change in the configuration file format".

I also think we need better guidance to point to than the one above. Maybe with a recipe/example. Either in this document or another document.

Comment (mat007) [reply desktop/enterprise/release-notes.md]: I added some step by step instructions on how to do the manual upgrade.
I tried various layouts and that one seemed like the least bad one, feel free to suggest something else!

**Comment by mat007** [comment]
> Weren't we going to put something at whatever URL the error message points to too?

I can do that as well, I’m not sure about how to work that in https://docs.docker.com/desktop/enterprise/admin/configure/windows-admin/ and https://docs.docker.com/desktop/enterprise/admin/configure/mac-admin/ as these pages are supposed to describe the latest version of the admin settings only at the moment.

**Comment by usha-mandya** [comment]
@mat007 @stephen-turner - How about creating a new section 'Configuration file format update' with this info in the Mac/Windows Admin topics? We could just point to these topics from the Known issues section to avoid repetition.

**Comment by mat007** [comment]
I’ve moved the manual update steps to the admin configuration pages and linked from the release notes.

**Pull request merged**